TeamCode/src/main/java/org/firstinspires/ftc/teamcode/DriveTrain.java

```diff
@@ -0,0 +1,646 @@
+
+import com.kauailabs.navx.ftc.AHRS;
+import com.qualcomm.robotcore.hardware.DcMotor;
+import com.qualcomm.robotcore.hardware.HardwareMap;
+import com.qualcomm.robotcore.hardware.Servo;
+import com.qualcomm.robotcore.util.Range;
+
+import org.firstinspires.ftc.robotcore.external.Telemetry;
+
+import static java.lang.Math.cos;
+import static java.lang.Math.sin;
+
+/**
+ * Created by pston on 10/22/2017
+ */
+
+public class DriveTrain {
+
+    /////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+// MOVE V
+    //objects for all the drive train motors
+    //objects for controlling the front two motors
+    private DcMotor leftFront;
+    //object for controlling the left front motor
+    private DcMotor rightFront;
+    //object for controlling the right front motor
+
+    //objects for controlling the rear two motors
+    private DcMotor leftRear;
+    //object for controlling the left rear motor
+    private DcMotor rightRear;
+
+    /////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+// GYRO V
+    private AHRS gyro;
+    private AHRS.DimStateTracker gyroReset;
+    //gyro sensor
+
+    //variables for gyro sensor
+    private float goalDegrees;
+    private int goalPosition;
+    private int gyroRange;
+    private static final double     countsPerMotorRev    = 1440 ;
+    private static final double     driveGearReduction    = 1.5 ;
+    private static final double     wheelDiameterInches   = 4.0 ;
+    private static final double     countsPerInch         = (countsPerMotorRev * driveGearReduction) / (wheelDiameterInches * Math.PI);
+    /////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+// SETUP V
+    private String LeftFront = "leftfront";
+    private String RightFront = "rightfront";
+    private String LeftRear = "leftrear";
+    private String RightRear = "rightrear";
+    private final int gyroPort = 3;
+    /////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+// MISS V
+    private Telemetry telemetry;        //do we need this?
+    //object for reference (telemetry)
```

Comment:
remove comment on line 57 and 58, add comment explaining why telemetry is defined